fiftyone/utils/cvat.py

```diff
@@ -5396,13 +5466,18 @@ def __init__(
         label_dict,
         class_map,
         attr_id_map,
+        server_id_map,
```

Comment:
You'll need to add `server_id_map = d.get("server_id_map", {})` to `_from_dict()` to ensure backwards compatibility of this new feature